GOVERNANCE.rst

```diff
@@ -167,22 +167,25 @@ New core developers can be nominated by any current core developer. Once
 they have been nominated, there will be a vote by the current core
 developers.
 
-Voting on new core developers is one of the few activities that takes
-place on the project’s private chat. While it is expected that most
-votes will be unanimous, a 2/3 majority of the cast votes is enough. The
-vote needs to be open for at least five working days.
+Voting on appointments is one of the few activities that takes
+place on the project’s private communication channels. The vote will be
+anonymous.
```

Comment:
one should also specify whether partial results should be visible while the vote is in progress. I'd say: no!

Reply:
Is there any app or so that allows to do that?

Reply:
polly seems to be able to do that, but it's not free: https://www.polly.ai/help/slack/advanced-options

Reply:
hm, ok - let´s not overcomplicate things for now, then.